engine/src/test/java/com/datatorrent/stram/engine/StreamingContainerTest.java

```diff
@@ -94,4 +127,14 @@ public void emitTuples()
     private static final Logger logger = LoggerFactory.getLogger(CommitAwareOperator.class);
```

Comment:
Change it to LoggerFactory.getLogger(StreamingContainerTest.class) and move to StreamingContainerTest.